### PR TITLE
lua-54 - new package (latest lua version)

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

**Comment by fritzkink** [comment]
BTW only 64bit.

File: components/runtime/lua-54/Makefile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+COMPONENT_ARCHIVE_URL_1= $(COMPONENT_PROJECT_URL)tests/$(COMPONENT_ARCHIVE_1)
+
+CLEAN_PATHS += $(LUA_TESTS)
+
+include $(WS_MAKE_RULES)/common.mk
+
+COMPONENT_BUILD_TARGETS= solaris
+
+# This is because lua uses $(CC) to build it's shared libraries.
+CC +=           $(CC_BITS)
```
Comment (AndWac) [inline components/runtime/lua-54/Makefile:56]: it's -> its

**Comment by AndWac** [comment]
Did you run "gmake test"?

**Comment by fritzkink** [comment]
Yes I ran gmake test, same result as with lua-53.

**Pull request merged**